config/noobs/partition_setup.sh

```diff
@@ -29,7 +29,7 @@ md5sumCheck() {
       echo "# LibreELEC failed md5 check - Installation will quit #"
       echo "#                                                     #"
       echo "#    Your original download was probably corrupt.     #"
-      echo "#  Please visit www.libreelec.tv and get another copy #"
+      echo "#  	Please visit libreelec.tv and get another copy 	  #"
```

Comment:
@InuSasha -> no, this seems to be a github diff visualisation issue as you can see in the full file: https://github.com/jo3rg/LibreELEC.tv/blob/changed_urls/config/noobs/partition_setup.sh#L32